### PR TITLE
Add more fields to floorist query

### PR DESCRIPTION
**Pull request by fijshion**
# Overview

This is based on feedback from the datahub team. This will be an incremental process to refine the query. The initial request is to send them data related to Red Hat products which I think this mostly covers.

## PR Checklist

- [ ] Keep PR title short, ideally under 72 characters
- [ ] Descriptive comments provided in complex code blocks
- [ ] Tests: validate optimal/expected output
- [ ] Tests: validate exceptions and failure scenarios
- [ ] Tests: edge cases
- [ ] Recovers or fails gracefully during potential resource outages (e.g. DB, Kafka)
- [ ] Documentation, if this PR changes the way other services interact with host inventory
- [ ] Links to related PRs

## Secure Coding Practices Documentation Reference

You can find documentation on this checklist [here](https://github.com/RedHatInsights/secure-coding-checklist).

## Secure Coding Checklist

- [ ] Input Validation
- [ ] Output Encoding
- [ ] Authentication and Password Management
- [ ] Session Management
- [ ] Access Control
- [ ] Cryptographic Practices
- [ ] Error Handling and Logging
- [ ] Data Protection
- [ ] Communication Security
- [ ] System Configuration
- [ ] Database Security
- [ ] File Management
- [ ] Memory Management
- [ ] General Coding Practices

### CODE REVIEW

**Review by kruai — DISMISSED** [review]
These fields all look correct to me.

**Comment by kruai** [comment]
I don't see any harm in merging, so I'm going to go ahead and do it. @ryandillinfelton @thearifismail feel free to review and lmk if you disagree

File: deployment.yaml [hunk 1]
```diff
@@ -648,10 +648,30 @@ objects:
...
+            "system_profile"->'sap_instance_number' AS "sap_hana_instance_number",
+            "system_profile"->'sap_version' AS "sap_hana_version",
+            "system_profile"->'host_type' AS "host_type",
+            "system_profile"->'system_purpose'->'usage' AS "system_purpose_usage",
+            "system_profile"->'system_purpose'->'role' AS "system_purpose_role",
+            "system_profile"->'system_purpose'->'sla' AS "system_purpose_sla",
+            "system_profile"->'ansible'->'controller_version' AS "ansible_controller_version",
+            "system_profile"->'ansible'->'hub_version' AS "ansible_hub_version",
+            "system_profile"->'ansible'->'catalog_worker_version' AS "catalog_worker_version",
+            "system_profile"->'ansible'->'sso_version' AS "sso_version",
```
Comment (ryandillinfelton) [inline deployment.yaml:668]: no ansible prefix for these ones?

**Review by ryandillinfelton — APPROVED** [review]
Left a comment pointing out what may be a naming discrepancy. If it's intended you've got the go ahead.

**Comment by fijshion** [comment]
good catch - force pushed an update

**Review by kruai — APPROVED** [review]

**Review by ryandillinfelton — APPROVED** [review]
LGTM

**Pull request merged**